src/Parser/WindwalkerMonk/Modules/Items/Katsuos.js

```diff
@@ -0,0 +1,34 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class Katsuos extends Module {
+  static dependencies = {
+    combatatants: Combatants,
+  };
+  chiSaved = 0; 
+  
+
+  on_initialized() {
+    this.active = this.combatatants.selected.hasFeet(ITEMS.KATSUOS_ECLIPSE.id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+
+    if (spellId === SPELLS.FISTS_OF_FURY_CAST.id &&! this.combatatants.selected.hasBuff(SPELLS.SERENITY_TALENT.id)) {
```

Comment:
Seeing as Serenity is actually a buff too, this should be 
```jsx
if (spellId === SPELLS.FISTS_OF_FURY_CAST.id && !this.combatants.selected.hasBuff(SPELLS.SERENITY_TALENT.id, event.timestamp))
```
As seen, please place the space between && and !
Also, are Fists of Fury the only Chi spender in WW?

Reply:
Katsuo's eclipse reduces the chi cost of Fists of Fury by 1, so it just counts the amount of Fists of Fury casts not made free by serenity.

Reply:
Oh wait, I misread the item's equip effect :D My bad, sorry ^^